Nur in bmp-extra-plugins-0.2.2.fix/input/musepack: tags.loT.
diff -ru bmp-extra-plugins-0.2.2/output/crossfade/monitor.c bmp-extra-plugins-0.2.2.fix/output/crossfade/monitor.c
--- bmp-extra-plugins-0.2.2/output/crossfade/monitor.c	2004-02-18 13:49:28.000000000 +0100
+++ bmp-extra-plugins-0.2.2.fix/output/crossfade/monitor.c	2008-08-19 20:42:44.000000000 +0200
@@ -35,9 +35,9 @@
 #include "support.h"
 
 
-static GtkWidget   *monitor_win;
-static GtkWidget   *monitor_display_drawingarea;
-static GtkProgress *monitor_output_progress;
+GtkWidget   *monitor_win;
+GtkWidget   *monitor_display_drawingarea;
+GtkProgress *monitor_output_progress;
 static GtkLabel    *monitor_position_label;
 static GtkLabel    *monitor_total_label;
 static GtkLabel    *monitor_left_label;
diff -ru bmp-extra-plugins-0.2.2/output/crossfade/oss.c bmp-extra-plugins-0.2.2.fix/output/crossfade/oss.c
--- bmp-extra-plugins-0.2.2/output/crossfade/oss.c	2004-01-13 10:27:04.000000000 +0100
+++ bmp-extra-plugins-0.2.2.fix/output/crossfade/oss.c	2008-08-19 20:43:15.000000000 +0200
@@ -94,7 +94,7 @@
 static gint     dsp_fragment_utime = 0;
 static gboolean dsp_select_works = FALSE;
 
-static pthread_mutex_t buffer_mutex;
+pthread_mutex_t buffer_mutex;
 static pthread_t       buffer_thread;
 
 static format_t  format;
